pkgs/tools/filesystems/squashfuse/default.nix

```diff
@@ -0,0 +1,65 @@
+{ stdenv, fetchurl, automake, autoconf, libtool, fuse, pkgconfig, pcre,
+
+# Optional Dependencies
+lz4 ? null, xz ? null, zlib ? null, lzo ? null, zstd ? null}:
```

Comment:
I’d change that to a list of supported compression algorithms

```
lz4, xz, zlib, lzo, zstd,
compressionAlgorithmSupport ? [ "lz4" "lzma" "zlib" "lzo" "zstd" ]`
```

Reply:
looks more complicated actually.

Reply:
> Also now nobody can now use the compressionAlgorithmSupport attribute in all-packages without colliding with this package.

I’m not sure I understand?